.github/workflows/graphql.yml

```diff
@@ -4,12 +4,12 @@ on:
     branches:
       - 'master'
     paths:
-      - 'internal/graphql/schema/schema.graphql'
+      - 'meshery/meshery'

```

Comment:
No @amino19 I wasn't talking about changing this. :sweat_smile:

Reply:
@DelusionalOptimist : Okay only `pull_request`, must be updated by `meshery/meshery`?

Reply:
@amino19 both `push` and `pull_request` events should be triggered only when `internal/graphql/schema/schema.graphql` changes.
That is, it was good the way it was earlier : )